pkg/kubelet/dockershim/docker_sandbox.go

```diff
@@ -276,7 +276,25 @@ func (ds *dockerService) StopPodSandbox(ctx context.Context, r *runtimeapi.StopP
 		return resp, nil
 	}
 
-	// TODO: Stop all running containers in the sandbox.
+	go func() {
+		opts := dockertypes.ContainerListOptions{All: true}
+
+		opts.Filters = dockerfilters.NewArgs()
+		f := newDockerFilter(&opts.Filters)
+		f.AddLabel(sandboxIDLabelKey, podSandboxID)
+
+		containers, err := ds.client.ListContainers(opts)
+		if err != nil {
+			klog.Errorf("Failed to list all container: %v", err)
+			return
+		}
+		for _, v := range containers {
+			err := ds.client.StopContainer(v.ID, defaultSandboxGracePeriod)
```

Comment:
Only invoke stop container for running ones here? For already stopped containers, I don't think we need to stop again.

Reply:
Updated. Only stop the running container.
Thanks